es.hjson

```diff
@@ -377,53 +377,53 @@
+  "joinconfig.title.help": "<code>/%s</code><b> le permite establecer el nombre del grupo con un mensaje.\n\nSu mensaje debería ser el siguiente:</b>\n<code>/%s Nuevo nombre del grupo o chat</code>\n\n<b>Sugerencia: </b><i>¿Tiene un nombre de grupo que siempre usa en los títulos? Echa un vistazo con</i>/cn"
+  "joinconfig.title.replyToTextHint": "<b>Si está respondiendo a otro mensaje para configurar el nombre del grupo, debe ser solo texto.</b>\n\n<b>Sugerencia: </b><i>¿Tiene un nombre del grupo que siempre usa en los títulos? Echa un vistazo con</i>/cn"
 
-  "joinconfig.chatname.help": "<code>/%s</code><b> le permite configurar el nombre del chat. Este nombre siempre aparecerá frente a cualquier título de chat configurado con </b><code>/t</code> <b>o</b><code>/title</code><b>.\n\nSu mensaje debería ser el siguiente:</b>\n<code>/%s MyChatName</code>"
-  "joinconfig.chatname.replyToTextHint": "<b>Si está respondiendo a otro mensaje para configurar el nombre del chat, debe ser solo texto.</b>"
-  "joinconfig.chatname.success": "<b>Su nombre de chat se ha configurado como:</b>\n%s\n\nEsto aparecerá al comienzo de cada conjunto de títulos de chat usando <code>/t</code> o <code>/title</code>."
+  "joinconfig.chatname.help": "<code>/%s</code><b> le permite configurar el nombre del grupo. Este nombre siempre aparecerá frente a cualquier nombre del grupo establecido con </b><code>/t</code> <b>o</b><code>/title</code><b>.\n\nSu mensaje debería ser el siguiente:</b>\n<code>/%s Nombre del grupo</code>"
+  "joinconfig.chatname.replyToTextHint": "<b>Si está respondiendo a otro mensaje para configurar el nombre del grupo, debe ser solo texto.</b>"
+  "joinconfig.chatname.success": "<b>El nombre del grupo se ha configurado como:</b>\n%s\n\nEsto aparecerá al inicio de cada nombre del grupo mediante el comando <code>/t</code> o <code>/title</code>."
 
 
   /* ---------------------------------------------------------------------
                         JoinVerify - Used for /JoinVerify
      --------------------------------------------------------------------- */
   # ⚠️ IMPORTANT
   "joinverify.callbacks": {
-    "correctAnswerText": "¡Correcto! El %s más para ir ..."
-    "incorrectAnswerText": "Incorrecto. 3 más para ir ..."
-    "success": "<b>¡Éxito!\n\nBuen trabajo humano, ahora puedes chatear en: </b><code>%s</code>"
-    "successAnswerText": "¡Éxito! Puedes chatear en %s ahora..."
-    "manuallyTrusted": "%s<b> ha confiado manualmente </b>%s<b>.</b>"
+    "correctAnswerText": "¡Correcto! Falta %s más para continuar ..."
+    "incorrectAnswerText": "Incorrecto. 3 más para continuar ..."
+    "success": "<b>¡Éxito!\n\nBuen trabajo, ahora puedes chatear en: </b><code>%s</code>"
+    "successAnswerText": "¡Éxito! Puedes entrar al grupo %s"

```

Comment:
If a group is using Join Verify (but not PreVerify), then the user will already be inside of the group.

I think it will be safer therefore to use "chat" instead of "enter", right?